msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -10,11 +10,13 @@ ms.date: 04/07/2022
 
 # Apps for Teams meetings and calls
 
-Meetings enable collaboration, partnership, informed communication, and shared feedback. The meeting app can deliver a user experience for each stage of the meeting lifecycle. Meeting lifecycle includes pre-meeting, in-meeting, and post-meeting app experience, depending on the attendee's status.
+Meetings enable collaboration, partnership, informed communication, and shared feedback. The meeting app can deliver a user experience for each stage of the meeting lifecycle. The following image gives you an idea of the meeting app extensibility:
```

Comment:
Suggestion, there might be some confusion on the Teams meeting app itself and separate 3P apps that interact with the meeting
```suggestion
Meetings enable collaboration, partnership, informed communication, and shared feedback. The meeting space can deliver a user experience for each stage of the meeting lifecycle. The following image gives you an idea of the meeting app extensibility:
```